### DISCUSSION TITLE
New Here

### DISCUSSION THREAD

**Discussion by IgnoreBox** [Authenticator-Extension/Authenticator#954] [Q&A] [answered]
first, i think this Addon is very simple 
so, we can trust this Addon, right?

now at Microsoft store, it says version 6.3.3 but at GitHub 6.3.4, how to update it?

**Comment by mymindstorm** [comment, marked as answer]
6.3.4 is just a version published to fix a policy compliance issue in Firefox and has no meaningful changes when compared to 6.3.3. That's why it has not been published on Edge.

**Reply by IgnoreBox** [reply]
I See... so i just wait on edge webstore for update ?
and this addon is safe right ? i cant find any review about this addon, but as we know... we just can "buy" review (so review is not 100% correct :p )

**Reply by mymindstorm** [reply]
Edge won't be getting the update since it is unnecessary for Edge. 

In terms of safety, we do our best to make the extension secure. You are free to review the source code of the extension and come to your own conclusions or compile it yourself if you do not trust the extension store or us.

**Reply by IgnoreBox** [reply]
well,,, i trust this addon... almost 100% source at github are safe ...

thankyou for your answer.. i hope this app/addon will keep in developtment and ..who knows. it will have Desktop apps :D


🍻
